perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -137,6 +137,41 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 
 	},
+
+	/*#
+	 * Send invitations with link to VO / Group application form from provided csv data
+	 *
+	 * If VO or Group have non-empty attribute urn:perun:[vo/group]:attribute-def:def:applicationURL
+	 * content is used as link to application form. Otherwise link is automatically generated based on
+	 * required AUTHZ in template and registrar url set in /etc/perun/perun-registrar.properties.
+	 *
+	 * General errors result in exception being thrown, single failures are skipped and added to result
+	 *
+	 * @param invitationData csv file values separated by semicolon ';'. Only [email; name] or [email] is valid format.
+	 * 		example: ["mail@mail.cz", "mail2@mail.cz;user2"]
+	 * @param vo VO to link form to
```

Comment:
`vo` param type should be `int` and not `VO` as we read it as integer in the code. Also I would prefer using `voId`, just like in standard `sendInvitation()` methods. Same goes for `group` param.